src/index.html

```diff
@@ -4,9 +4,389 @@
+
+          <div class="card__review review">
+            <div class="review__stars stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="review__reviews">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-title">Price:</p>
+            <p class="card__cost">$2,199</p>
+          </div>
+
+          <button
+            data-qa="card-hover"
+            type="button"
+            class="card__buy-batton"
+          >
+          Buy
+          </button>
+        </div>
+      </article>
+
+
+      <article data-qa="card" class="card">
+        <div class="card__container">
+          <div class="card__product-photo">
+            <img
+              src="images/imac.jpeg"
+              alt="APPLE iMac 27"
+              class="card__img"
+            >
+          </div>
+
+          <h2 class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+          <p class="card__product-code">Product code: 195434</p>
+
+          <div class="card__review review">
+            <div class="review__stars stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="review__reviews">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-title">Price:</p>
+            <p class="card__cost">$2,199</p>
+          </div>
+
+          <button
+            data-qa="card-hover"
+            type="button"
+            class="card__buy-batton"
+          >
+          Buy
+          </button>
+        </div>
+      </article>
+
+
+      <article data-qa="card" class="card">
+        <div class="card__container">
+          <div class="card__product-photo">
+            <img
+              src="images/imac.jpeg"
+              alt="APPLE iMac 27"
+              class="card__img"
+            >
+          </div>
+
+          <h2 class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+          <p class="card__product-code">Product code: 195434</p>
+
+          <div class="card__review review">
+            <div class="review__stars stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="review__reviews">Reviews: 5</p>
+          </div>
```

Comment:
```suggestion
          <div class="card__review-wrapper">
            <div class="card__stars stars stars--4">
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
            </div>

            <p class="card__reviews">Reviews: 5</p>
          </div>
```
don't create a new `review` BEM block
One BEM element can be nested inside another BEM element, and in this time both of them are elements of the same BEM block